### ISSUE TITLE
Mermaid diagrams not correctly initialized/cleaned

### ISSUE THREAD

**Issue by RomanKornev** [notable/notable#430]
- **OS Version**: Win10
- **Notable Version**: v1.3.0

Originated in #313 but new version changed the behavior.
#### 1. Diagrams disappear on note switch
Create a note:

    ```mermaid
    gantt
    dateFormat  YYYY-MM-DD
    title Adding GANTT diagram to mermaid

    section A section
    Completed task            :done,    des1, 2014-01-06,2014-01-08
    Active task               :active,  des2, 2014-01-09, 3d
    Future task               :         des3, after des2, 5d
    Future task2               :         des4, after des3, 5d
    ```
Duplicate it
Switch to new one and change it slightly
Switch back
Diagram is missing on both notes
Using Force Reload fixes it

#### 2. Git graph adds duplicates of itself
Create a note:

    ```mermaid
    gitGraph:
    options
    {
        "nodeSpacing": 80,
        "nodeRadius": 10
    }
    end
    commit
    branch newbranch
    checkout newbranch
    commit
    commit
    checkout master
    commit
    commit
    merge newbranch
    ```

Switch to another note
Switch back
Git graph is now twice the size and contains twice the commits
Using Force Reload fixes it

#### 3. ClassDiagram only works once
Create a note:

    ```mermaid
    classDiagram
    Class01 <|-- AveryLongClass : Cool
    Class03 *-- Class04
    Class05 o-- Class06
    Class07 .. Class08
    Class09 --> C2 : Where am i?
    Class09 --* C3
    Class09 --|> Class07
    Class07 : equals()
    Class07 : Object[] elementData
    Class01 : size()
    Class01 : int chimp
    Class01 : int gorilla
    Class08 <--> C2: Cool label
    ```
Class diagram appeared just fine
Switch to another note
Switch back
Class diagram error: `[mermaid error: Cannot read property 'x' of undefined]`
Using Force Reload fixes it

### Expected behavior

1. Diagrams are supposed to be preserved on note switch
2. Make sure Git Graph doesn't duplicate itself on note switch
3. Ensure ClassDiagram works after note switch

**Comment by fabiospampinato** [comment]
> 1. Diagrams disappear on note switch
> 3. ClassDiagram only works once

I think this parts (maybe the other one too) might be a duplicate of #621.

**Comment by fabiospampinato** [comment]
Fixed in v1.6.
